src/pyff/utils.py

```diff
@@ -273,7 +273,13 @@ def load_url(url, enable_cache=True, timeout=60):
     if url.startswith('file://'):
         path = url[7:]
         if not os.path.exists(path):
-            raise IOError("file not found: %s" % path)
+            log.error("file not found: %s" % path)
+            return _Resource(result=None,
```

Comment:
This is in line with something else I've been thinking about: remove the retry decorator and have HTTP errors generate "empty" resources instead of doing retrys in the inner http loader loop. Opinions?